.circleci/config.yml

```diff
@@ -23,7 +24,8 @@ jobs:
     steps:
       - checkout
       - run: sudo apt-get install unzip
-      - run: . ./install-prototool.sh
+      - run: . ./install-buf.sh
+      - run: . ./install-protoc.sh
```

Comment:
Is the `- run: go install github.com/golang/protobuf/protoc-gen-go` line below this block still necessary as it's imported in `tools.go`?

Reply:
Yes, the `tools.go` definition just adds the dependency to our modules dependency tree, we still need to install it to execute it.